pricing-page-exercise/salih/styles.css

```diff
@@ -0,0 +1,195 @@
+/* Global */
+
+*, ::after, ::before {
+    box-sizing: border-box;
+    font-family:sans-serif;
+}
+ul{
```

Comment:
It's ok to use `ul` selector in this case but it would be safer to use a class instead to select only the lists you need to change style for.